blockchain/txvalidator.go

```diff
@@ -273,12 +273,17 @@ func (b *BlockChain) CheckTransactionContext(blockHeight uint32,
 		}
 		return nil
 
+	case CRCProposalRealWithdraw:
+		if err := b.checkCRCProposalRealWithdrawTransaction(txn, references); err != nil {
+			log.Warn("[checkCRAssetsRectifyTransaction],", err)
```

Comment:
log.warn should be  "checkCRCProposalRealWithdrawTransaction" err

Reply:
fixed